content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/aging.rst

```diff
@@ -0,0 +1,96 @@
+======================
+Inventory aging report
+======================
+
+.. |SVLs| replace:: :abbr:`SVLs (stock valuation layers)`
+
+The inventory aging report evaluates all items in stock, providing insights into purchasing costs
+and potential profitability.
+
+Create customized pivot tables to analyze product, operation types, month, or company breakdowns.
+This helps identify products that stay rotting in stock.
+
+.. note::
+   The reporting feature is only accessible to users with :doc:`admin access
```

Comment:
Q: is it this report specifically, or the whole **Reporting** menu inside _Inventory_? Consider being more specific in this note 😉

Reply:
```suggestion
   The :guilabel:`Reporting` menu in *Inventory* is only accessible to users with :doc:`admin access
```